src/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -44,7 +44,7 @@ See the LICENSE file in the project root for more information.
       <CppCompilerAndLinkerArg Include="-I $(IlcPath)/inc" />
       <CppCompilerAndLinkerArg Condition="'$(Configuration)' == 'Debug'" Include="-g -O0" />
       <CppCompilerAndLinkerArg Condition="'$(Configuration)' != 'Debug'" Include="-O2" />
-      <CppCompilerAndLinkerArg Include="-c -Wno-invalid-offsetof" />
+      <CppCompilerAndLinkerArg Include="-c -Wno-invalid-offsetof -fms-extensions" />
```

Comment:
We try to generate highly portable C/C++ in CppCodeGen. Depending on ms-extensions is not very portable.

Could please add macro for this https://github.com/dotnet/corert/blob/master/src/Native/Bootstrap/CppCodeGen.h instead, and not use `-fms-extensions` here?

Reply:
Thank you! Done.